chainerx_cc/chainerx/routines/activation.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include <cstdint>
+
+#include <nonstd/optional.hpp>
+
+#include "chainerx/array.h"
+#include "chainerx/constant.h"
```

Comment:
You can remove this include, as well as `cstdint` and `nonstd/optional.hpp` but please add `scalar.h`.

Reply:
Sure! Thanks for pointing that out.